client/my-sites/plans-v2/utils.ts

```diff
@@ -688,12 +688,12 @@ export function checkout(
 	if ( ! siteSlug ) {
 		path = `/jetpack/connect/${ productsString }`;
 	} else {
-		path = isJetpackCloud()
+		path = isJetpackCloudProd()
 			? `/checkout/${ siteSlug }/${ productsString }`
 			: `/checkout/${ siteSlug }`;
 	}
 
-	if ( isJetpackCloud() ) {
+	if ( isJetpackCloudProd() ) {
```

Comment:
Confused: what's going on here? Do we want to disable these for non-prod Cloud?

Reply:
@ChaosExAnima, in Cloud, except for prod, we'd like the same behaviour as in Calypso, i.e. redirecting to `/checkout/` in the same platform.